server/sandbox_stop_linux.go

```diff
@@ -84,6 +86,12 @@ func (s *Server) stopPodSandbox(ctx context.Context, sb *sandbox.Sandbox) error
 
 	log.Infof(ctx, "Stopped pod sandbox: %s", sb.ID())
 	sb.SetStopped(true)
+	if s.config.EventedPLEG {
+		if err := s.Runtime().UpdateContainerStatus(ctx, sb.InfraContainer()); err != nil {
```

Comment:
This is called during the stop logic, so not sure why we are adding a call here.

Reply:
I agree, will report here after testing if we really need to have this `UpdateContainerStatus` during stop/remove